android/gradle.properties

```diff
@@ -54,7 +54,7 @@ commitHash=unknown
 # to write custom TurboModules/Fabric components OR use libraries that
 # are providing them.
 newArchEnabled=false
-
+ORG_GRADLE_PROJECT_universalApk=false
```

Comment:
needed for `make nix-update-gradle` this env var then becomes undefined.
Probably could be moved to `Makefile`